From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Danny Lin <danny@kdrag0n.dev>
Date: Wed, 6 Oct 2021 18:40:30 -0700
Subject: [PATCH 11/24] SystemUI: Make QS panel change on UI mode changes

This reverts commit d62f7249f9e3222da95ecf6816601c408aac6be5.

This is a prerequisite for making the QS panel follow the light system
theme setting.

Change-Id: Iac4c96ccb3845812ca3df820bf27dc533816f72e
Signed-off-by: Saalim Quadri <danascape@gmail.com>
(cherry picked from commit d15f6b23a2287d7a1a9055b57ec906830c0078d1)
---
 packages/SystemUI/res/values/styles.xml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/packages/SystemUI/res/values/styles.xml b/packages/SystemUI/res/values/styles.xml
index 5bbebc918d81..b69e8009428f 100644
--- a/packages/SystemUI/res/values/styles.xml
+++ b/packages/SystemUI/res/values/styles.xml
@@ -516,7 +516,7 @@
         <item name="containerStyle">@style/AuthNonBioCredentialPinPasswordContainerStyle</item>
     </style>
 
-    <style name="Theme.SystemUI.QuickSettings" parent="@*android:style/Theme.DeviceDefault">
+    <style name="Theme.SystemUI.QuickSettings" parent="@*android:style/Theme.DeviceDefault.SystemUI">
         <item name="isQsTheme">true</item>
         <item name="lightIconTheme">@style/QSIconTheme</item>
         <item name="darkIconTheme">@style/QSIconTheme</item>
